### PR TITLE
chore: Add import related eslint rules

### PR DESCRIPTION
**Pull request by foxyblocks**
- Turns on lint rules for "import/order" and "no-unused-imports".
- Also ran `eslint --fix` on the entire
  codebase to automatically apply the new rules

The only file that was manually changed in this PR is

### CODE REVIEW

File: .eslintrc.json
Comment (foxyblocks) [inline .eslintrc.json:1]: This is the only file that was manually changed in this PR everything else was updated programatically

**Comment by brandonroberts** [comment]
@foxyblocks looks like the formatter didn't fix all the occurrences of the imports

**Comment by foxyblocks** [comment]
> @foxyblocks looks like the formatter didn't fix all the occurrences of the imports

Yah, it looks like some changes got merged into the PR branch that the new lint rule hadn't had a chance to run on yet. It's all resolved now

**Review by brandonroberts — APPROVED** [review]
🧹

**Pull request merged**